api/data_overview_builder.py

```diff
@@ -21,23 +21,35 @@ class Fields:
     metrics: List[str]
 
 
-CASES_FIELDS = Fields(["cases", "newCases"], ["caseDensity", "infectionRate", "infectionRateCI90"],)
+CASES_FIELDS = Fields(
+    ["cases", "newCases"],
+    ["caseDensity", "infectionRate", "infectionRateCI90", "weeklyNewCasesPer100k"],
+)
 
 DEATHS_FIELDS = Fields(["deaths"], [])
 
-HOSPITALIZATIONS_FIELDS = Fields(["icuBeds", "hospitalBeds"], ["icuCapacityRatio"],)
+HOSPITALIZATIONS_FIELDS = Fields(
```

Comment:
I don't think I realized this in the past, but this builds the output of `api/docs/docs/data-definitions.md`. I didn't add the CDC community metrics here when we added them, and instead added them manually. A bit more on this below...